src/components/Button/src/Button.vue

```diff
@@ -126,10 +141,57 @@ function ghost(tokens) {
 	};
 }
 
+function success(tokens) {
```

Comment:
ok there's gotta be a less copy-pasty way to do this right? this is just `fill()` with hardcoded `color` and `textColor`.

Reply:
Probably worth creating a util function that spits out these values holistically. `getFillColor`, or something to that effect